GitHubDashboard/Controllers/QueryController.cs

```diff
@@ -54,11 +54,15 @@ public QueryController(GitHubClient gitHubClient)
 
         private async Task<IReadOnlyList<Issue>> GetIssuesAsync(string owner, string repository, string milestone, string labels)
         {
-            // "undefined" means that milestone was not set on the incoming URL.  Something like this:
+            // Map the incoming values from the URL to values that Octokit can process.  There are some quirks that need
+            // clarification.  "undefined" means that milestone was not set on the incoming URL.  Something like this:
             //          http://<host>/count/nuget/home?label=VS1ES
-            // Octokit treats milestone = '*' as "any set milestone" as opposed to "any milestone
-            // including no milestones."  So, if an issue had a label but no milestone setting, and query
-            // included "milestone = '*'" - Octokit (nor the website) would not return it.
+            // Map that value (and null/whitespace) to NULL in the Octokit issue request.  This tells Octokit "don't
```

Comment:
I still don't love this, because no part of the C# code ever sees a URL like the above.  That's what the angular app sees in TS, but the C# code only gets the path separated components.  This comment makes it seem like the C# code is handling that URL.